src/footer/js/footer-breadcrumbs.js

```diff
@@ -1,7 +1,8 @@
 import {$assign as $} from '../../shared/js/shared';
 
 export default (data) => {
-  const showBreadCrumbs = data.showBreadcrumb;
+  // const showBreadCrumbs = data.showBreadcrumb;
+  const showBreadCrumbs = true;
```

Comment:
it looks like you hard coded to enable breadcrumbs, can you set that back to reading the variable